frontend/tailwind.config.js

```diff
@@ -5,7 +5,11 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        "screen-minus-h20": "calc(100vh - 80px)",
```

Comment:
[imo]
「h20って何の高さだっけ？」ってなりそうだから、「screen-without-header」とかの方が情報量が適切だと思った！
```suggestion
        "screen-without-header": "calc(100vh - 80px)",
```

Reply:
修正しました